systemimage/device.py

```diff
@@ -45,6 +45,11 @@ def get_device(self):
             log.exception('getprop exit status: {}'.format(error.returncode))
             return '?'
         except FileNotFoundError as error:
-            log.exception('getprop command not found')
-            return '?'
+            # Try to use device-info instead
+            log.exception('getprop command not found, trying fallback to deviceinfo')
+            try:
+                stdout = check_output(
+                    'device-info get Name'.split(), universal_newlines=True)
+            except:
+                return '?'
```

Comment:
You must specify an exception to catch in Python. This is a syntax error.

Reply:
I dont think so, this should match all exceptions?

Reply:
See https://stackoverflow.com/questions/4990718/about-catching-any-exception